### PR TITLE
fix(device_info_plus): Updated the description of getAndroidId method 

### PR DESCRIPTION
**Pull request by martinoyovo**
## Description

*Added the correct description about getAndroidId method*

## Related Issues

*https://github.com/fluttercommunity/plus_plugins/issues/769*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated the version in `pubspec.yaml` and `CHANGELOG.md`.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

File: packages/device_info_plus/device_info_plus/android/src/main/java/dev/fluttercommunity/plus/device_info/MethodCallHandlerImpl.java [hunk 1]
```diff
@@ -98,10 +98,8 @@ private String[] getSystemFeatures() {
   }
 
   /**
-   * Returns the Android hardware device ID that is unique between the device + user and app
-   * signing. This key will change if the app is uninstalled or its data is cleared. Device factory
-   * reset will also result in a value change.
-   *
+   * Returns the Android hardware device ID that is unique between the device + user and app signing.
+   * This key will change if a factory reset is performed on the device or if an APK signing key changes.
```
Comment (vbuberen) [inline packages/device_info_plus/device_info_plus/android/src/main/java/dev/fluttercommunity/plus/device_info/MethodCallHandlerImpl.java:102]: I think it is still not fully correct and we should also mention the difference for devices with Android versions before 8 and those with 8 and higher.

Suggest something like:
```
On devices with Android >= 8.0 (API level 26) returns an ID that is unique to each combination of app-signing key, user, and device. The value may change if a factory reset is performed on the device or if an APK signing key changes.
On devices with Android < 8.0 (API level 26) returns an ID that is randomly generated when the user first sets up the device and should remain constant for the lifetime of the user's device.
```

**Review by vbuberen — CHANGES_REQUESTED** [review]
Thanks for your contribution. Added a suggestion on improving the change.

Would ask you to update the PR title to comply with our contributors guide https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md#55-create-a-pull-request

File: packages/device_info_plus/device_info_plus/android/src/main/java/dev/fluttercommunity/plus/device_info/MethodCallHandlerImpl.java (hunk 1 above)
Comment (vbuberen) [reply packages/device_info_plus/device_info_plus/android/src/main/java/dev/fluttercommunity/plus/device_info/MethodCallHandlerImpl.java:102]: @martinoyovo You might have missed this comment ☝🏻

**Comment by vbuberen** [comment]
Thanks for your contirbution.

**Review by vbuberen — APPROVED** [review]

**Comment by a7medpa** [comment]
FYI: I installed version 3.2.4 and in the generated_plugin.registrant.dart file, I got this error: Target of URI doesn't exist: 'package:device_info_plus_web/device_info_plus_web.dart'

The error goes when I downgrade back to 3.2.3

**Pull request merged**